mungegithub/mungegithub.go

```diff
@@ -25,12 +25,13 @@ import (
 	"k8s.io/contrib/mungegithub/features"
 	github_util "k8s.io/contrib/mungegithub/github"
 	"k8s.io/contrib/mungegithub/mungers"
+	"k8s.io/contrib/mungegithub/mungers/mungerutil"
 	"k8s.io/contrib/mungegithub/reports"
 	utilflag "k8s.io/kubernetes/pkg/util/flag"
 
+	"github.com/foxish/viper"
```

Comment:
There is a discussion in the comments. There was a patch to replace the TOML library that was being used (as it is under the WTFPL), but it was reverted. @bgrant0607 suggested that we've waited long enough and should just fork it.